core-iconset.html

```diff
@@ -187,13 +202,13 @@
   },
 
   updateThemes: function () {
-    this.lightDom.querySelectorAll('property[theme]')
+    Polymer.dom.querySelectorAll('property[theme]', this)
```

Comment:
I realize this isn't your code, but `<property>` is not a real element and doesn't seem to be registered anywhere as a custom element.